intdataplane/int_dataplane.go

```diff
@@ -591,6 +606,31 @@ func (d *InternalDataplane) apply() {
 		}
 		d.cleanupPending = false
 	}
+
+	// Set up any needed rescheduling kick.  Any pending kick will have been recalculated above
+	// so we always cancel and reschedule.
```

Comment:
I was trying to explain that we don't need a queue of ticks because the iptables.Table objects will recalculate any rescheduling that they need each time.